features/step_definitions/tournament.rb

```diff
@@ -115,3 +115,34 @@ def placing_to_display_string(placing)
   visit event_path single_tournament
   expect(page).to have_text("#{placing_to_display_string placing} #{team.name}")
 end
+
+Then(/^the standing of the (home|away) team of match (\w+) (\d+) links to (\w+) (\d+)$/) do |home_or_away, match_gameday, match_num, target_match_gameday, target_match_num|
+  visit event_schedule_path single_tournament
+  match = find_match_on_page target_match_gameday, target_match_num
+  team = find_team_of_match match_gameday, match_num, home_or_away
+  visit event_overview_path single_tournament
+  within(:xpath, "//table/tbody/tr[contains(./td/a, '#{team.name}')]") do
+    expect(page).to have_link(href: match_path(match))
+  end
+end
+
+Then(/^the opponent of the (home|away) team of match (\w+) (\d+) links to the (home|away) team of match (\w+) (\d+)$/) do |home_or_away, match_gameday, match_num, home_or_away_opponent, target_match_gameday, target_match_num|
```

Comment:
these two step definitions seem to have quite a lot of common code. Can you extract some of it to methods?

Reply:
Considering the local variables in the code, sadly not.